imagevector/images.yaml

```diff
@@ -16,6 +16,12 @@ images:
     name: resource-manager
   sourceRepository: github.com/gardener/gardener
   repository: eu.gcr.io/gardener-project/gardener/resource-manager
+- name: gardener-metrics-exporter
+  resourceId:
+    name: metrics-exporter
+  sourceRepository: github.com/gardener/gardener-metrics-exporter
+  repository: eu.gcr.io/gardener-project/gardener/metrics-exporter
+  tag: "0.27.0"
```

Comment:
Please move this somewhere below L162 - we only enlist the core Gardener images here